benchbuild/utils/run.py

```diff
@@ -66,8 +66,8 @@ def __begin(self, command: BaseCommand, project, experiment, group):
             (run, session), where run is the generated run instance and
             session the associated transaction for later use.
         """
-        from benchbuild.utils.db import create_run
         from benchbuild.utils import schema as s
+        from benchbuild.utils.db import create_run
```

Comment:
Ignore this warning for now. Moving this to  the top-level means initializing the database early. Not a good idea.